tests/test_end_to_end.py

```diff
@@ -350,3 +351,10 @@ def test_builtin_plugins(tmp_path):
 def test_missing_object_exception():
     invalid_page_id = "11111111111111111111111111111111"
     assert run_n2y([invalid_page_id]) != 0
+
+
+def test_comment():
+    block_with_comments_id = "ac496c7db16743488495976f7433dbfb"
```

Comment:
How do page-level comments work? Do those come from the `ChildPageBlock`?

Reply:
Yes, page comments just come from it's block (ie: `page._block`)